src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMedrecReloading.java

```diff
@@ -77,18 +77,18 @@ private boolean isModuleVisible(AdsBaseObject ads, By moduleSelector) {
   private void scrollAndWait(Duration duration) {
     JavascriptActions jsActions = new JavascriptActions(driver);
 
-    PageObjectLogging.log("User action", String.format("Scroll %d", 200), true);
+    Log.log("User action", String.format("Scroll %d", 200), true);
     jsActions.scrollBy(0, 200);
 
     waitSomeTime(duration);
 
-    PageObjectLogging.log("User action", String.format("Scroll %d", -200), true);
+    Log.log("User action", String.format("Scroll %d", -200), true);
     jsActions.scrollBy(0, -200);
   }
 
   private void waitSomeTime(Duration reloadDelay) {
     try {
-      PageObjectLogging.log("Sleep", String.format("Wait for %d ms", reloadDelay.toMillis()), true);
+      Log.log("Sleep", String.format("Wait for %d ms", reloadDelay.toMillis()), true);
       Thread.sleep(reloadDelay.toMillis());
     } catch (InterruptedException ignored) {}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00108)